server/status/feed.go

```diff
@@ -91,6 +92,7 @@ func (nef NodeEventFeed) CallComplete(args proto.Request, reply proto.Response)
 // listen for events published by nodes.
 type NodeEventListener interface {
 	OnStartNode(event *StartNodeEvent)
+	OnTrace(event *tracer.Trace)
```

Comment:
This is fine for now, but can you add a TODO to break this out into its own "TraceEventListener"?  Actually, i'm trying to figure out if this is the best way to handler these listeners; as is, you can't use multiple event listeners on the same subscription.